example.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+use Utopia\Detector\Detection\Framework\Astro;
+use Utopia\Detector\Detection\Framework\Flutter;
+use Utopia\Detector\Detection\Framework\NextJs;
+use Utopia\Detector\Detection\Framework\Nuxt;
+use Utopia\Detector\Detection\Framework\Remix;
+use Utopia\Detector\Detection\Framework\SvelteKit;
+use Utopia\Detector\Detection\Packager\Npm;
+use Utopia\Detector\Detection\Packager\Pnpm;
+use Utopia\Detector\Detection\Packager\Yarn;
+use Utopia\Detector\Detection\Runtime\Bun;
+use Utopia\Detector\Detection\Runtime\Cpp;
+use Utopia\Detector\Detection\Runtime\Dart;
+use Utopia\Detector\Detection\Runtime\Deno;
+use Utopia\Detector\Detection\Runtime\Dotnet;
+use Utopia\Detector\Detection\Runtime\Java;
+use Utopia\Detector\Detection\Runtime\Node;
+use Utopia\Detector\Detection\Runtime\PHP;
+use Utopia\Detector\Detection\Runtime\Python;
+use Utopia\Detector\Detection\Runtime\Ruby;
+use Utopia\Detector\Detection\Runtime\Swift;
+use Utopia\Detector\Detector\Framework;
+use Utopia\Detector\Detector\Packager;
+use Utopia\Detector\Detector\Rendering;
+use Utopia\Detector\Detector\Runtime;
+use Utopia\Detector\Strategy;
+
+include_once 'vendor/autoload.php';
+
+// $files = ['src/main.js', '.gitignore', 'package.json', 'yarn.lock'];
+$files = ['bun.lockb', 'fly.toml', 'package.json', 'remix.config.js'];
+
+/**
+ * Function flows
+ */
+
+// 1. Detect NPM
+$detector = new Packager($files);
+$detector
+    ->addOption(new Pnpm)
```

Comment:
I think we can just add all this options by default in the constructor of the Packager class. No real need to set them up externally each time we need an instance.

Reply:
As discussed, we might want to enable different runtimes/frameworks for cloud/self-hosted and thus, keeping the change for now